tests/Tests.cs

```diff
@@ -25,11 +25,13 @@ public class ControllerTests
 
 			// assert
 			var viewResult = Assert.IsType<ViewResult>(result);
-			Assert.Equal(expected: expectedDefaultUrl, actual: viewResult.ViewData["url"] as String);
-			Assert.Equal(expected: expectedDefaultType, actual: viewResult.ViewData["type"] as String);
-			Assert.Equal(expected: expectedDefaultTitle, actual: viewResult.ViewData["title"] as String);
-			Assert.Equal(expected: expectedDefaultDescription, actual: viewResult.ViewData["description"] as String);
-			Assert.Equal(expected: expectedDefaultImage, actual: viewResult.ViewData["image"] as String);
+			var viewModel = viewResult.Model as MetaViewModel;
+
+			Assert.Equal(expected: expectedDefaultUrl, actual: viewModel.url as String);
+			Assert.Equal(expected: expectedDefaultType, actual: viewModel.type as String);
+			Assert.Equal(expected: expectedDefaultTitle, actual: viewModel.title as String);
+			Assert.Equal(expected: expectedDefaultDescription, actual: viewModel.description as String);
+			Assert.Equal(expected: expectedDefaultImage, actual: viewModel.image as String);
```

Comment:
Style: The `as String` can be removed from all of these now that we have a strongly typed model.